SNIPS/snip-75.md

```diff
@@ -0,0 +1,213 @@
+- **Phone Call** - goes through silent mode (using Opus Genie).
+- **Telegram**
+- **Email**
+
+**Note**: In an emergency response, Committee members should be capable of responding with a laptop and a hard ledger wallet during drills involving a realistic emergency scenario.
+
+#### Scope of Work/Services
+Assess and improve StarkGate’s monitoring and security system, including:
+- Reviewing the monitoring service triggers for alerts.
+- Ensuring that all essential assets are monitored.
+- Analyzing StarkWare's monitoring in comparison with other services and chains.
+- Proactively surveying the monitoring agent's dashboard and event detection to manually identify suspicious activities and suggest alert improvements.
+
+#### Work Hours
+The Committee member is required to be on call 24/7. If unavailable on any given day, they must notify the council administrator at least 24 hours in advance. **Days of non-availability are not limited.**
+
+### Starknet Governing Body Code of Conduct
+To foster a community of respect, inclusivity, and safety, all members of the Council are expected to adhere to the following principles:
+- **Active Participation**: Engage in discussions and decision-making.
+- **Accessibility**: Remain accessible to the community.
+- **Ethical Behavior**: Uphold high standards in all interactions.
+- **Positive Communication**: Foster a respectful culture.
+- **Respect and Non-Discrimination**: Promote diversity and inclusivity.
+- **Non-Violence**: Avoid promoting violence.
+- **No Abuse of Position**: Avoid exploiting their position.
+- **Conflict of Interests**: Declare and manage conflicts of interest.
+- **Confidentiality**: Protect sensitive information.
+
+### Adding and Removing Members
+The Starknet Foundation is responsible for appointing and compensating Council members from an administrative perspective. Members may be removed based on the following criteria:
+1. Failure to adhere to the code of conduct.
+2. Failure to undertake duties and responsibilities.
+3. A conflict of interest arises.
+4. Responsibility for a security breach.
+5. Loss of technical or strategic knowledge.
+6. Community loss of confidence.
+
+### Security Council Phasing In
+
+#### **Phase 1**
+In phase 1, the Security Council will monitor upgrades affecting the Starknet staking core infrastructure, including:
+
+**On Starknet**:
+- Staking
+- Operator
+- Minting curve
+- L2 rewards supply
+- Delegation pool
+
+**On Ethereum**:
+- STRK mint manager
+- L1 reward supplier
+
+Additionally, the SC will act as the:
+- **Security Admin on the L2 Staking contract**:
+  - Setting the security agent (the entity that can pause the contracts)
+  - Unpausing the contracts
+- **Security Admin on L1 STRK mint manager**:
+  - Setting the security agent (the entity that can revoke allowance)
+- **Security Agent on L1**:
+  - Revoking allowance in case of emergency
```

Comment:
revoking allowance of what contract

Reply:
The security agent is the entity that can pause the affected contract. 
See the pausing flow https://github.com/Eikix/SNIPs/blob/security-council/SNIPS/snip-75.md#pausing-flow